scripts/ci/wasm_unpack.js

```diff
@@ -0,0 +1,235 @@
+(function(source, uncompressedSize) {
+	var base64DecToArr = (Buffer !== undefined && typeof Buffer.from === "function") ?
```

Comment:
Not entirely sure if it wasn't safer to just always use the JS implementation, nor if this is a sane way to check for a suitable ``Buffer.from`` to be available... might be more reasonable to be conservative and forgo the speedup this brings on node...

Reply:
I mean... it's 10% speedup, it's not nothing :-). But yeah, we're talking like 50ms per load, so it's probably really not worth it.

Reply:
How? I see 4 seconds of difference in sys+user. Unless you're speaking about the wall-clock time?

Reply:
Yeah, in sys+user it's ~4 seconds with a bit over a hundred seconds in total - but that's still a few percent or so - 10% is only real-time/wall-clock time. But that's over a course of 100 loads, so in any case, it's a negligible amount of time per load.